src/main/java/com/infoshareacademy/service/RecipeService.java

```diff
@@ -0,0 +1,63 @@
+package com.infoshareacademy.service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.infoshareacademy.fridge.Fridge;
+import com.infoshareacademy.recipe.Recipe;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class RecipeService {
+
+    public void writeJson(List<Recipe> recipe) throws IOException {
+        Path path = Path.of("src", "resources", "recipe.json");
+        File file = new File(path.toString());
+        ObjectMapper objectMapper = new ObjectMapper();
+        ObjectWriter objectWriter = objectMapper.writerWithDefaultPrettyPrinter();
+        objectWriter.writeValue(file, recipe);
+    }
+
+    public List<Recipe> getJson() throws IOException {
+        Path path = Path.of("src", "resources", "recipe.json");
+        ObjectMapper objectMapper = new ObjectMapper();
+        File file = new File(path.toString());
+        List<Recipe> recipe = objectMapper.readValue(file, new TypeReference<List<Recipe>>() {
+        });
+        return recipe;
+    }
```

Comment:
Duplikacja kodu jak w klasie FridgeService. Warto pomyśleć, aby na przyszłość to zmienić.